tests/functional/test_manage.py

```diff
@@ -0,0 +1,35 @@
+
+import json
+import os
+import requests
+import time
+
+
+def test_manage_make_migrations(test_project):
+    test_project.run_cmd('testproject/manage.py makemigrations initial')
+
+    assert '001_initial.py' in os.listdir('testproject/migrations')
+
+
+def test_manage_migrate(test_project):
+    test_project.run_cmd('testproject/manage.py makemigrations initial')
+    test_project.run_cmd('testproject/manage.py migrate')
+    test_project.run_cmd('sqlite3 testproject.db .tables | grep user')
+
+
+def test_manage_runserver(test_project):
+    test_project.run_cmd('testproject/manage.py makemigrations initial')
+    test_project.run_cmd('testproject/manage.py migrate')
+
+    process = test_project.run_popen('testproject/manage.py runserver')
```

Comment:
Why not `run_cmd` ? Do we need `runserver` to be non-blocking?

Reply:
`run_cmd` runs a command and blocks until it exists. We need the server running while performing our test request and then we want to stop it. The simplest way to achieve that is by forking using `subprocess.Popen`.

Reply:
Ahh, of course. My bad.